script-library/common-debian.sh

```diff
@@ -70,6 +70,8 @@ apt-get-update-if-needed()
 
 # Run install apt-utils to avoid debconf warning then verify presence of other common developer tools and dependencies
 if [ "${PACKAGES_ALREADY_INSTALLED}" != "true" ]; then
+    # Adding non-free packges on Debian
+    echo -e "deb http://deb.debian.org/debian/ $(lsb_release -cs) main contrib non-free \ndeb http://deb.debian.org/debian/ $(lsb_release -cs)-updates main contrib non-free \ndeb http://deb.debian.org/debian-security/ $(lsb_release -cs)/updates main contrib non-free \ndeb http://deb.debian.org/debian/ $(lsb_release -cs)-backports main contrib non-free" >/etc/apt/sources.list
```

Comment:
The `$(lsb_release -cs)` command is executed multiple times here. It might be nice to execute it once before running this, capture the output to a variable, and then use that variable here.

Reply:
Added a variable `CODENAME`